.mvn/wrapper/maven-wrapper.properties

```diff
@@ -1,2 +0,0 @@
-distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.8.5/apache-maven-3.8.5-bin.zip
```

Comment:
Lets keep the maven-wrapper.properties file but exclude the maven-wrapper.jar binary from the project.

* Run `mvn wrapper:wrapper -Dmaven=3.8.6` in project root
* fix .gitignore file for `maven-wrapper.jar`, so that git ignores the jar.